lib/Dialect/TritonGPU/Transforms/Utility.cpp

```diff
@@ -676,13 +676,21 @@ getConvertBackwardSlice(Value root, SetVector<Value> &slice,
                         Attribute rootEncoding,
                         DenseMap<Value, Attribute> &layout,
                         std::function<bool(Operation *)> stopPropagation) {
-  DenseSet<Value> visited;
-  SmallVector<std::pair<Value, Attribute>> queue = {{root, rootEncoding}};
+  DenseSet<std::pair<Value, Attribute>> seen;
```

Comment:
Is it better to just use a SetVector?

Reply:
Pairs remain in seen after they've been popped off the queue.